test-apps/hello-world-modules/src/main/java/io/github/pixee/testapp/Main.java

```diff
@@ -0,0 +1,15 @@
+package io.github.pixee.testapp;
+
+import io.github.pixee.security.HostValidator;
+
+public final class Main {
+
```

Comment:
Maybe a comment here?
```suggestion
/** A simple piece of code that references the library, so we know the module visibility is correct. */
```